### ISSUE TITLE
[suggestion] - change the “keybag” system

### ISSUE THREAD

**Issue by Mesrine67** [SOH69/mm_carkeys#9]
using the container system could improve

https://overextended.dev/ox_inventory/Guides/creatingItems#creating-container-items

**Comment by SOH69** [comment]
the system i am using works on ox, QB, esx combine
this thing will only work for ox so I have to make diff for other inv

**Comment by Mesrine67** [comment]
no, I've already tested them on qbx and esx, but you'll have to check whether they save on disconnection.
